providers/base/bin/sleep_test_log_check.py

```diff
@@ -126,13 +133,20 @@ def main():
                               provide a list of UNIQUE errors encountered in \
                               the log file. It will not display duplicates. \
                               Default is [%(default)s]")
-    parser.add_argument('test',
+    parser.add_argument('-t', '--test',
                         action='store',
-                        choices=['s3', 's4'],
-                        help='The test to check (s3 or s4)')
+                        default='all',
+                        help="The test (ie.s3 ,s4 ,klog ,oope ,...etc) \
+                              to check. Default is [%(default)s]")
```

Comment:
You should explain that the `all` option is a special entry to check everything at once. (at least that's what I understand from the code)